databuilder/rest_api/rest_api_query.py

```diff
@@ -56,7 +56,8 @@ def __init__(self,
                  fail_no_result=False,  # type: bool
                  skip_no_result=False,  # type: bool
                  json_path_contains_or=False,  # type: bool
-                 can_skip_failure=None,  # type: Callable
+                 can_skip_failure=None,  # type: Callable,
+                 **kwargs  # type: Any
```

Comment:
do we need this kwargs ?

Reply:
Yeah, it's because of subclass's below line. It passes more than it supports and throwing error without **kwargs.

`super(ModePaginatedRestApiQuery, self).__init__(**kwargs)`